descriptors/ModuleDescriptor-template.json

```diff
@@ -527,6 +531,10 @@
       "id": "order-lines",
       "version": "3.0"
     },
+    {
```

Comment:
Looks like will be circulation dependency between mod-invoice and mod-orders

Reply:
Thanks. I am not sure there would be a circular dependency because invoice is optional in mod-orders. But I think you are right anyway that orders should be optional in mod-invoice. I moved that to optional.